katharsis-core/src/main/java/io/katharsis/jackson/serializer/LinkageContainerSerializer.java

```diff
@@ -23,45 +20,37 @@
     private static final String TYPE_FIELD_NAME = "type";
     private static final String ID_FIELD_NAME = "id";
 
-    private final ResourceRegistry resourceRegistry;
+    public LinkageContainerSerializer() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)